app/javascript/dashboard/i18n/locale/en/settings.json

```diff
@@ -63,13 +63,27 @@
       "AUDIO_NOTIFICATIONS_SECTION": {
         "TITLE": "Audio Notifications",
         "NOTE": "Enable audio notifications in dashboard for new messages and conversations.",
+        "PLAY": "Play sound",
         "ALERT_TYPES": {
           "NONE": "None",
           "MINE": "Assigned",
-          "ALL": "All"
+          "ALL": "All",
+          "ASSIGNED": "My assigned conversations",
+          "UNATTENDED": "New conversations needing attention",
+          "NOTME": "Open conversations assigned to everyone else"
+        },
+        "ALERT_COMBINATIONS": {
+          "NONE": "You won't receive any audio alerts",
+          "ASSIGNED": "You'll receive alerts for conversations assigned to you",
+          "UNATTENDED": "You'll receive alerts for any unattended conversations",
+          "NOTME": "You'll receive alerts for conversations assigned to others",
+          "ASSIGNED+UNATTENDED": "You'll receive alerts for your assigned conversations and any unattended ones",
+          "ASSIGNED+NOTME": "You'll receive alerts for conversations assigned to your both yours and others",
```

Comment:
Doesn't Assigned + not me include all conversations?

Reply:
It won't include unassigned conversations